ompi/mca/osc/monitoring/Makefile.am

```diff
@@ -32,8 +32,11 @@ mcacomponent_LTLIBRARIES = $(component_install)
 mca_osc_monitoring_la_SOURCES = $(monitoring_sources)
 mca_osc_monitoring_la_LDFLAGS = -module -avoid-version
 mca_osc_monitoring_la_LIBADD = $(top_builddir)/ompi/lib@OMPI_LIBMPI_NAME@.la \
-	$(OMPI_TOP_BUILDDIR)/ompi/mca/common/monitoring/libmca_common_monitoring.la
+		$(OMPI_TOP_BUILDDIR)/ompi/mca/common/monitoring/libmca_common_monitoring.la
 
 noinst_LTLIBRARIES = $(component_noinst)
 libmca_osc_monitoring_la_SOURCES = $(monitoring_sources)
 libmca_osc_monitoring_la_LDFLAGS = -module -avoid-version
+
+distclean-local:
+	rm -f $(builddir)/osc_monitoring_template_gen.h
```

Comment:
Minor nit: you can add this file to `DISTCLEANFILES` and not have to make a `distclean-local` rule.